dvc/remote/gdrive.py

```diff
@@ -46,7 +46,12 @@ def should_retry(exc):
             return False
 
         retry_codes = [403, 500, 502, 503, 504]
-        return exc.error.get("code", 0) in retry_codes
+        result = exc.error.get("code", 0) in retry_codes
+        if result:
+            logger.debug(
+                "Retry GDrive API call failed with {}".format(str(exc))
```

Comment:
not sure you need `str( )` here (cc @efiop - please TAL)

Reply:
right, seems, is not needed. Have seen such usage elsewhere in DVC.